org-code-javabuilder/playground/src/main/java/org/code/playground/Board.java

```diff
@@ -15,7 +15,6 @@
 
   private boolean firstRunStarted;
   private boolean isRunning;
-  private boolean hasEnded;
```

Comment:
I removed the use of this in [this PR](https://github.com/code-dot-org/javabuilder/pull/130/files), cleaning it up now